openedx/core/djangolib/testing/utils.py

```diff
@@ -101,7 +101,7 @@ def start_cache_isolation(cls):
 
         cls.__old_settings.append(copy.deepcopy(settings.CACHES))
         override = override_settings(CACHES=cache_settings)
-        override.__enter__()
+        override.__enter__()  # pylint: disable=unnecessary-dunder-call
```

Comment:
This also causes the unit tests to fail upon refactoring. Detailed Error Stack Trace: https://github.com/openedx/edx-platform/actions/runs/3265468332/jobs/5367724316#step:7:3517